scss/_variables.scss

```
@@ -1735,31 +1738,35 @@ $tooltip-arrow-color:               null !default; // Deprecated in Boosted 5.2.
 // Popovers
 
 // scss-docs-start popover-variables
-$popover-font-size:                 $font-size-sm !default;
+$popover-font-size:                 $font-size-base !default; // Boosted mod
 $popover-font-weight:               $font-weight-bold !default; // Boosted mod
-$popover-bg:                        $gray-400 !default; // Boosted mod: instead of `var(--#{$prefix}body-bg)`
-$popover-max-width:                 17.25rem !default;
+$popover-bg:                        var(--#{$prefix}body-bg) !default;
+$popover-max-width:                 23.75rem !default; // Boosted mod
```

```suggestion
$popover-max-width:                 $spacer * 19 !default; // Boosted mod
```

Wondering which one is best here

---

I think it could be better to keep the non hardcoded value for now.